src/enums/entityTypes.ts

```diff
@@ -1,4 +1,49 @@
+/** Entity types (aka corp type codes in Colin). */
 export enum EntityTypes {
+  BC_CCC = 'CC',
+  BC_COMPANY = 'BCC', // FUTURE: should be 'BC'
+  BC_CORPORATION = 'CR', // NOT IN COLIN - NR ONLY?
+  BC_ULC_COMPANY = 'ULC',
+  BENEFIT_COMPANY = 'BEN',
+  CCC_CONTINUE_IN = 'CCC',
+  CO_1860 = 'QA',
+  CO_1862 = 'QB',
+  CO_1878 = 'QC',
+  CO_1890 = 'QD',
+  CO_1897 = 'QE',
+  CONT_IN_SOCIETY = 'CS',
+  CONTINUE_IN = 'C',
   COOP = 'CP',
-  BCOMP = 'BEN'
+  CEMETARY = 'CEM',
+  EXTRA_PRO_A = 'A',
+  EXTRA_PRO_B = 'B',
+  EXTRA_PRO_REG = 'EPR',
+  FINANCIAL = 'FI',
+  FOREIGN = 'FOR',
+  LIBRARY = 'LIB',
+  LICENSED = 'LIC',
+  LIM_PARTNERSHIP = 'LP',
+  LIMITED_CO = 'LLC',
+  LL_PARTNERSHIP = 'LL',
+  MISC_FIRM = 'MF',
+  PARISHES = 'PAR',
+  PARTNERSHIP = 'GP',
+  PENS_FUND_SOC = 'PFS',
+  PRIVATE_ACT = 'PA',
+  RAILWAYS = 'RLY',
+  REGISTRATION = 'REG',
+  SOCIETY = 'S',
+  SOCIETY_BRANCH = 'SB',
+  SOLE_PROP = 'SP',
+  TRAMWAYS = 'TMY',
+  TRUST = 'T',
+  ULC_CONTINUE_IN = 'CUL',
+  ULC_CO_1860 = 'UQA',
+  ULC_CO_1862 = 'UQB',
+  ULC_CO_1878 = 'UQC',
+  ULC_CO_1890 = 'UQD',
+  ULC_CO_1897 = 'UQE',
+  XPRO_COOP = 'XCP',
+  XPRO_SOCIETY = 'XS',
+  XPRO_LL_PARTNR = 'XL',
```

Comment:
wowsa, comprehensive list! Did we need all of these?

Reply:
No, not yet. But the list is in the BE and we will probably, ultimately, make use of these.

Reply:
We've got a lot of work ahead of us then 😅